doc/tinydb/Home.md

```diff
@@ -0,0 +1,69 @@
+# Welcome to the TinyDB!
+TinyDB is a course designed to help you quickly familiarize yourself with the underlying storage engine of [TiKV Project](https://github.com/tikv/tikv). 
```

Comment:
How about regarding TinyDB as one lab of TinyKV? and I think `TinyEngine` would be less ambiguous.

Reply:
Thanks for your suggestion. I agree `TinyEngine` is more appropriate. I will modify the description in the next submission.